app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -80,6 +83,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View rootView = inflater.inflate(R.layout.fragment_memory_graph, container, false);
         ((MemoryGraphParent) getActivity()).setActionBarTitle(getResources().getString(R.string.mem_graph));
         memGraph = rootView.findViewById(R.id.mem_graph);
+        if (getArguments().getString(LOG_FILE_KEY) != null) {
+            filePath = getArguments().getString(LOG_FILE_KEY);
+            Log.d("File Path received: ", "" + filePath);
```

Comment:
@CloudyPadmal Have a look here. The filePath received is perfectly fine : `/storage/emulated/0/NeuroLab/abcd.csv`. But while carrying out the `importData` function, it seems to show the FileNotFoundException :thinking: Why do you think that is? Can you check this once on your side? I took care of the perms which are granted. If you find something please tell me, till then I am trying different ways.

Reply:
The LoggerActivity and other functionality works fine. It is just about the above exception. :confused:

Reply:
Let me check

Reply:
@CloudyPadmal Good news! Have fixed the bug. Will push latest changes.